get-docker-lang-versions.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+printf 'Docker image,PHP version,NodeJS version,Python version\n' >> docker-lang-versions.txt
+
+while IFS= read -r line; do
```

Comment:
I prefer this sort of loop:

while read -r TABLE; do
   # code  using $TABLE
done < <(command that generates the input)

IFS playing with end of line/record markers in the shell. Things get ugly if things go wrong.

Reply:
How does that work for simply reading each line of the input piped to this shell script? I'm not very familiar with loops in bash scripts. Feel free to try out improvements locally and suggest code changes here to match a better way you've found.

Reply:
It doesn't so much. I'm just pointing it out.